examples/library_folders/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(LIBRARY_FOLDERS_TARGET library_folders)
+
+file(GLOB library_folders_sources *.c)
+add_executable("${LIBRARY_FOLDERS_TARGET}" ${library_folders_sources})
```

Comment:
To make `make` not automatically build this target, use [`EXCLUDE_FROM_ALL`](https://cmake.org/cmake/help/v3.0/command/add_executable.html).